src/App.css

```diff
@@ -58,4 +63,44 @@ button {
 button:hover {
   font-size: 23px;
   box-shadow: 1px 1px 2px black;
-}
\ No newline at end of file
+  cursor: pointer;
+}
+
+.frequencyButtons {
+  margin-top: 5vh;
+}
+
+.frequencyButtons input[type="radio"] {
```

Comment:
I've never seen this in CSS before, thanks for teaching me something new!

Reply:
Attribute selectors are super handy and powerful! You can match on partial strings even!